app/controllers/DocumentController.js

```diff
@@ -162,36 +160,36 @@ export default class DocControl {
   }
 
   /**
-    * This method get all documents in the systems
-    *
-    * @param {Object} date
-    * @param {Array} documents
-    * @returns {Boolean} true or false
-    */
+   * This method get all documents in the systems
+   *
+   * @param {Object} date
+   * @param {Array} documents
+   * @returns {Boolean} true or false
+   */
   dateFilter(date, documents) {
     return documents.filter(document =>
       date === JSON.stringify(document.createdAt).substr(1, 10));
   }
 
   /**
-    * This method get all documents in the systems
-    *
-    * @param {String} role
-    * @param {Array} documents
-    * @returns {Object} true or false
-    */
+   * This method get all documents in the systems
+   *
+   * @param {String} role
+   * @param {Array} documents
+   * @returns {Object} true or false
+   */
   roleFilter(role, documents) {
     return documents.filter(document =>
       role.toLowerCase() === document.Role.title.toLowerCase());
   }
 
   /**
-    * This method get all documents in the systems
-    *
-    * @param {Object} query
-    * @param {Array} docs
-    * @returns {Array} true or false
-    */
+   * This method get all documents in the systems
```

Comment:
###Nitpick

Pay attention to write your comment phrases, clauses and sentences in their correct grammatical form.